### DISCUSSION TITLE
Nemo - allow bulky to rename a single file

### DISCUSSION THREAD

**Discussion by Jeremy7701** [linuxmint/discussions#530] [Ideas]
Currently `Bulky` can be used to rename multiple files, in a variety of ways - such as changing case etc.

If I want to use its features for a single file, I need to create a scrap file, then select the two 'multiple' files.
This is very clumsy.

This could almost be offered as a `nemo action`.
I might give it a try!

**Comment by rcalixte** [comment]
Why would you need a separate tool or custom action to rename a single file?

**Comment by Jeremy7701** [comment]
Essentially for the same reason as you would want a tool to rename multiple files.
Perhaps to replace multiple occurences of spaces with hyphens or to fix the case etc of files.

**Comment by clefebvre** [comment]
You can run bulky on its own and then add/open the file from it. You'd be faster to just rename the file directly though.

**Reply by Jeremy7701** [reply]
Thanks - I'll do that!
